src/api/v1/db/oracledb/setupDb.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE PIZZA_INGREDIENTS;
+DROP TABLE PIZZAS;
+DROP TABLE INGREDIENTS;
+DROP TABLE DOUGHS;
+
+CREATE TABLE "DOUGHS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL, 
+	"GRAMS_FLOUR" NUMBER(*,0) NOT NULL, 
+	"GRAMS_WATER" NUMBER(*,0) NOT NULL, 
+	"FLOUR_TYPE" VARCHAR2(256 BYTE) NOT NULL, 
+	"WATER_TEMP" NUMBER(*,0) NOT NULL, 
+	"GRAMS_YEAST" NUMBER(*,0) NOT NULL, 
+	"GRAMS_SALT" NUMBER(*,0) NOT NULL, 
+	"GRAMS_SUGAR" NUMBER(*,0) NOT NULL, 
+	"GRAMS_OLIVE_OIL" NUMBER(*,0) NOT NULL, 
+	"BULK_FERMENT_TIME" NUMBER(*,0) NOT NULL, 
+	"PROOF_TIME" NUMBER(*,0) NOT NULL, 
+	"SPECIAL_INSTRUCTIONS" VARCHAR2(512 BYTE), 
+	 PRIMARY KEY ("ID")
+   );
+
+CREATE TABLE "INGREDIENTS" 
+   (	"TYPE" VARCHAR2(20 BYTE) NOT NULL, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL, 
+	"NOTES" VARCHAR2(256 BYTE), 
+	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL, 
```

Comment:
You don't need to set a min and max value for ID https://github.com/osu-mist/game-curator-api/blob/4929a92be94da94b9fb2084382360c9832da033b/api/v1/db/oracledb/setupDB.sql#L6

Reply:
Thanks, it looks like the cache stuff is optional as well